src/scripts/main.js

```diff
@@ -22,7 +22,20 @@ const food = {
 const tree = document.querySelector('#tree');
 
 function createTree(element, data) {
-  // WRITE YOUR CODE HERE
+  const list = document.createElement('ul');
+
+  element.append(list);
+
+  for (const key in data) {
+    const listItem = document.createElement('li');
+
+    listItem.innerText = key;
+    list.append(listItem);
+
+    if (Object.keys(data[key]).length !== 0) {
```

Comment:
```suggestion
    if (Object.keys(data[key]).length) {
```
Just a little simplification

Reply:
Code with this condition looks more understandable for me)